internal/nginx-meshctl/deploy/deploy_test.go

```diff
@@ -43,7 +44,41 @@ var _ = Describe("Deploy", func() {
 		// we need to set either tracing/telemetry to nil
 		values.Telemetry = nil
 		values.Tracing = nil
-		Expect(deployer.Deploy()).To(Succeed())
+		_, err := deployer.Deploy()
+		Expect(err).To(Succeed())
+	})
+
+	It("deploys air-gapped mesh", func() {
+		// mute stdout for this test since it will print the entire manifest
+		stdout := os.Stdout
+		defer func() { os.Stdout = stdout }()
+		os.Stdout = os.NewFile(0, os.DevNull)
+
+		var values helm.Values
+		Expect(yaml.Unmarshal(valuesYaml, &values)).To(Succeed())
+		deployer.Values = &values
+
+		// default deploy values are not valid
+		// we need to set either tracing/telemetry to nil
```

Comment:
one teeny-tiny little nit: if we have to set both to nil please specify "both" instead of "either"